__tests__/main.test.ts

```diff
@@ -46,20 +45,19 @@ describe('main.ts tests', () => {
         expect(AzureSqlAction).toHaveBeenCalled();
         expect(detectIPAddressSpy).toHaveBeenCalled();
         expect(getAuthorizerSpy).not.toHaveBeenCalled();
-        expect(getInputSpy).toHaveBeenCalledTimes(4);
+        expect(getInputSpy).toHaveBeenCalledTimes(3);
```

Comment:
Can you please provide some insights over this change?

Reply:
The input doesn't need to be called anymore to get the server-name as now is returned as part of the connection string object, hence it requires one less call